schema.graphql

```diff
@@ -281,40 +282,43 @@ type SubscriptionModule @entity {
   feeAmount: BigInt!
   feeToken: Bytes!
   periodDuration: BigInt!
-  prePaymentPeriods: BigInt!
-  resumePrePaidPeriods: BigInt!
-  latePaymentPenaltyPct: BigInt!
   governorSharePct: BigInt!
-  totalDonated: BigInt!
-  totalPaid: BigInt!
-  totalCollected: BigInt!
-  totalGovernorShares: BigInt!
   periods: [SubscriptionPeriod!] @derivedFrom(field: "instance")
-  subscribers: [Subscriber!] @derivedFrom(field: "instance")
+  tokens: [SubscriptionToken!] @derivedFrom(field: "instance")
```

Comment:
what do you think about calling these `SubscriptionBalance` instead?

Reply:
I wouldn’t remove the “Token” part, because I think it’s important to emphasize that this is the relationship between Subscription and Token. So maybe “SubscriptionTokenBalance”?

Reply:
`SubscriptionTokenBalance` sounds good :)